main/windows/index.ts

```diff
@@ -175,15 +172,15 @@ export class Tray {
         const gasDisplay = Math.round(hexToInt(gasPrice) / 1000000000).toString()
         title = gasDisplay // ɢ 🄶 Ⓖ ᴳᵂᴱᴵ
       }
-      setSystemTrayTitle(title)
+      systemTray?.setTitle(title)
     })
     this.readyHandler = () => {
-      initSystemTray(this.toggle)
-      setContextMenu('hide', menuClickHandlers, getDisplaySummonShortcut())
+      this.ready = true
+      systemTray = new SystemTray(this, menuClickHandlers)
```

Comment:
this line can  be moved to the constructor, then line 175 doesn't need a `?` check